pkg/database/vercode.go

```diff
@@ -195,6 +195,16 @@ func (db *Database) FindVerificationCodeByUUID(uuid string) (*VerificationCode,
 	return &vc, nil
 }
 
+// ExpireCode saves a verification code as expired.
+func (db *Database) ExpireCode(uuid string) (time.Time, error) {
+	now := time.Now().UTC()
+	vc := VerificationCode{
+		ExpiresAt:     now,
+		LongExpiresAt: now,
+	}
+	return now, db.db.Model(&vc).Where("uuid = ?", uuid).Debug().Update(&vc).Error
```

Comment:
.Debug?

also - I still think should make sure it makes sense to expire the code in a Tx:
like: https://github.com/google/exposure-notifications-verification-server/blob/main/pkg/database/token.go#L169

the code shouldn't have already been claimed and it shouldn't be already expired.

Reply:
you can also check those w/ a single SQL statement